site/content/installation/installing-nic/create-license-secret.md

```diff
@@ -0,0 +1,149 @@
+
+{{%tab name="Manifests"%}}
+
+Specify the Secret name in the `license-token-secret-name` Management ConfigMap key.
+
+For detailed guidance on creating the Management ConfigMap, refer to the [Management ConfigMap Resource Documentation]({{< relref "configuration/global-configuration/mgmt-configmap-resource/" >}}).
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+**If you are reporting to the default licensing endpoint, then you can now proceed with [installing NGINX Ingress Controller]({{< relref "installation/installing-nic/" >}}). Otherwise, follow the steps below to configure reporting to NGINX Instance Manager.**
+
+
+### Reporting to NGINX Instance Manager {#nim}
+
+If you are deploying NGINX Ingress Controller in an "air-gapped" environment you will need to report to NGINX Instance Manager (NIM) instead of the default licensing endpoint.
+
+First, you must specify the endpoint of your NGINX Instance Manager:
+
+{{<tabs name="nim-endpoint">}}
+
+{{%tab name="Helm"%}}
+
+Specify the endpoint using the `controller.mgmt.usageReport.endpoint` helm value.
+
+{{% /tab %}}
+
+{{%tab name="Manifests"%}}
+
+Specify the endpoint in the `usage-report-endpoint` Management ConfigMap key.
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+#### Configuring SSL Certificates and SSL Trusted Certificates {#nim-cert}
+
+To configure SSL Certificates and/or SSL Trusted Certificate, a few extra steps are needed.
+
+To use Client Auth with NIM, first create a Secret of type `kubernetes.io/tls` in the Namespace that the NIC Pod(s) are in. 
+
+```shell
+kubectl create secret tls ssl-certificate --cert=<path-to-your-client.pem> --key=<path-to-your-client.key> -n <Your Namespace>
+```
+
+To provide a SSL Trusted Certificate, and an optional Certificate Revocation List, create a Secret of type `nginx.org/ca` in the Namespace that the NIC Pod(s) are in.
+
+```shell
+kubectl create secret generic ssl-trusted-certificate \
+   --from-file=ca.crt=<path-to-your-ca.crt> \
+   --from-file=ca.crl=<path-to-your-ca.crl> \ # optional
+   --type=nginx.org/ca
+```
+
+Providing an optional CRL will configure the [`ssl_crl`](https://nginx.org/en/docs/ngx_mgmt_module.html#ssl_crl) directive.
+
+{{<tabs name="nim-secret-install">}}
+
+{{%tab name="Helm"%}}
+
+Specify the SSL Certificate Secret name using the `controller.mgmt.sslCertificateSecretName` helm value. 
+
+Specify the SSL Trusted Certificate Secret name using the `controller.mgmt.sslTrustedCertificateSecretName` helm value.
+
+{{% /tab %}}
+
+{{%tab name="Manifests"%}}
+
+Specify the SSL Certificate Secret name in the `ssl-certificate-secret-name` Management ConfigMap key.
+
+Specify the SSL Trusted Certificate Secret name in the `ssl-trusted-certificate-secret-name` Management ConfigMap key.
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+<br>
+
+**Once these Secrets are created and configuration options are set, you can now [install NGINX Ingress Controller ]({{< relref "installation/installing-nic" >}}).**
+
+## What’s reported and how it’s protected {#telemetry}
```

Comment:
Is this the same information as https://docs.nginx.com/nginx-ingress-controller/overview/product-telemetry/ ? If so, I'd update that page.

It seems like it would be best to have this information somewhere more visible and reference it from the document, rather than have it exist purely here.